src/plugins/common/list/lib/list-plugin.js

```diff
@@ -322,6 +327,24 @@ define([
 			}
 		},
 
+		/**
+		 * Default classes to apply to lists and list items
+		 */
+		defaultClasses: {
+			ul: {
```

Comment:
Add the option to provide a string rather than an array of strings, which is then internally normalized into and array of a single string.